### ISSUE TITLE
J_top overlapping features

### ISSUE THREAD

**Issue by sedorr** [andreisperid/E-TKT#29]
On the J_top-Part there is a screw hole hitting a countersink.
![PXL_20230418_190135623](https://user-images.githubusercontent.com/34986602/232881898-f210ec80-fd21-46e1-8aa4-116cd1ecc8d4.jpg)
![Screenshot 2023-04-18 210710](https://user-images.githubusercontent.com/34986602/232882851-bccf5ecc-9d68-4463-b50d-6803bae9b002.png)

**Comment by andreisperid** [comment]
Yes, I have noticed that!

It is not ideal, but the display holder has a little chamfer to avoid interference:

<img width="926" alt="image" src="https://user-images.githubusercontent.com/15098003/232886525-273b4463-3223-44ca-ba06-92813ca736da.png">

I decided to keep it that way to avoid major redesign, while keeping the structural capability for the display holder. I could fill in the gap, but it would be so thin it wouldn't make a relevant difference.
